assets/resourceSnippets/Automanage Best Practices DevTest Assignment.snippet.json

```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "metadata": {
+        "prefix": "arm-automanage-best-practices-devtest-assignment",
+        "description": "Create a Best Practices DevTest Configuration Profile Assignment"
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Compute/virtualMachines/providers/configurationProfileAssignments",
```

Comment:
@bmoore-msft Is it better to use a scope property like the following that I compiled from the bicep in https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/scope-extension-resources#apply-to-resource?
```json
    {
      "type": "Microsoft.Authorization/locks",
      "apiVersion": "2016-09-01",
      "scope": "[format('Microsoft.Storage/storageAccounts/{0}', 'examplestore')]",
      "name": "storeLock",
      "properties": {
        "level": "CanNotDelete",
        "notes": "Storage account should not be deleted."
      }
    }

```

Reply:
using `scope` is the pattern we've been migrating to...

Reply:
@StephenWeatherford Ive applied the scope property and verified that it deploys correctly. Does it look correct to you? 

- Automanage Best Practices DevTest Assignment.snippet.json
- Automanage Best Practices Prod Assignment.snippet.json
- Automanage Create Custom Profile.snippet.json